articles/articles/helper/simpleschema.php

```diff
@@ -60,13 +62,13 @@ public function mapPost($blog, $strip_tags='', $text_length=0, $skip=array())
 		else
 		{
 			//get media from custom component
-			$media_obj = new TZ_PortfolioModelMedia();
-			$med = $media_obj->getMedia($blog->id);
+			// $media_obj = new TZ_PortfolioModelMedia();
```

Comment:
@vinayak-patil Why we have commented this code? What was it used for?

Reply:
@ankush-maherwal Hi, This code is used to format the response, to get media(image) from 3rd party extension TZ_Portfolio.

So if we are accessing articles using latest resource (option=com_api&app=articles&resource=latest) and if TZ_Portfolio extension is not exist then we are getting error '0 Class "TZ_PortfolioModelMedia" not found'

Reply:
Hi @ankush-maherwal , As this is related to TZ_Portfolio extension and I don't know the use and working of this extension so I am restoring this code.